src/fsharp/MethodOverrides.fs

```diff
@@ -40,10 +41,24 @@ type OverrideInfo =
     member x.ReturnType = let (Override(_, _, _, _, _, b, _, _)) = x in b
     member x.IsCompilerGenerated = let (Override(_, _, _, _, _, _, _, b)) = x in b
 
-// If the bool is true then the slot is optional, i.e. is an interface slot
-// which does not _have_ to be implemented, because an inherited implementation 
-// is available.
-type RequiredSlot = RequiredSlot of MethInfo * (* isOptional: *) bool 
+[<System.Flags>]
+type RequiredSlotFlags =
+    | None                                      = 0x0000
```

Comment:
What's the significance of this value (and others)? If these raw values have some intrinsic meaning, can we get a comment indicating that?

Reply:
These are just flags. The values themselves do not represent any significance other than flipping a bit to determine if something is true or false.